gcovr/tests/test_gcovr.py

```diff
@@ -32,42 +32,60 @@
 from pyutilib.misc.pyyaml_util import compare_repn as compare_xml
 from pyutilib.misc.xmltodict import parse as parse_xml
 
-python_interpreter = sys.executable.replace('\\', '/')  # use forward slash on windows as well
+python_interpreter = sys.executable.replace(
+    "\\", "/"
+)  # use forward slash on windows as well
 env = os.environ
-env['GCOVR'] = python_interpreter + ' -m gcovr'
+env["GCOVR"] = python_interpreter + " -m gcovr"
+for var in [
+    "CPATH",
+    "C_INCLUDE_PATH",
+    "CPLUS_INCLUDE_PATH",
+    "OBJC_INCLUDE_PATH",
+    "CFLAGS",
+    "CXXFLAGS",
+    "LDFLAGS",
+    "LANG"
+]:
+    if var in env:
+        env.pop(var)
```

Comment:
Instead of *deleting* LANG, it could make sense to force it to a standard locale such as `LANG=C` or `LANG=C.UTF-8`. We already do that in the Dockerfile since `black` seemed to have problems otherwise, cf #487. Either way, LGTM.

Reply:
You're right. Will change this before merge.